flasc/data_processing/dataframe_manipulations.py

```diff
@@ -965,8 +1094,7 @@ def sun_alt(row):
 
 
 def plot_sun_altitude_with_day_night_color(df: pd.DataFrame, ax: plt.axis = None):
-    """
-    Plot Sun Altitude with Day-Night Color Differentiation.
+    """Plot Sun Altitude with Day-Night Color Differentiation.
```

Comment:
Not sure why this one has capitalized letters?

Reply:
It's a very exciting function?  Fixed